jenkins/bootstrap_test.py

```diff
@@ -982,7 +984,12 @@ def testOnlyJobs(self):
             self.assertTrue(os.access(job_path, os.X_OK|os.R_OK), job_path)
 
     def testAllProjectAreUnique(self):
-        projects = collections.defaultdict(list)
+        allowed_list = {  # TODO(fejta): remove these
```

Comment:
I want to commit with this... I don't like that there are multiple jobs reusing the same project, but for I'm not going to try and change the behavior of these jobs right now.